walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -192,9 +194,9 @@ public void doubleBookedPeople() {
     MeetingRequest request = new MeetingRequest(Arrays.asList(PERSON_A), DURATION_30_MINUTES);
 
     Collection<TimeRange> actual = query.query(events, request);
-    Collection<TimeRange> expected =
-        Arrays.asList(TimeRange.fromStartEnd(TimeRange.START_OF_DAY, TIME_0830AM, false),
-            TimeRange.fromStartEnd(TIME_0930AM, TimeRange.END_OF_DAY, true));
+    Collection<TimeRange> expected = Arrays.asList(
+        TimeRange.fromStartEnd(TimeRange.START_OF_DAY, TIME_0830AM, /*inclusive=*/false),
+        TimeRange.fromStartEnd(TIME_0930AM, TimeRange.END_OF_DAY, /*inclusive=*/true));
```

Comment:
Not sure the /*inclusive=*/ tags are entirely necessary, but I guess they're nice to have.

Reply:
For ambiguous arguments like true or false, they should be used (especially in tests). It improves readability drastically.

Reply:
Yeah I think I remember seeing this originally in one of Keiffer's PRs. I agree that it is much more readable this way so I will definitely keep it.